weave

```diff
@@ -1634,6 +1634,7 @@ case "$COMMAND" in
         protect_against_docker_hang
         VOLUME_CONTAINERS=$(docker ps -qa --filter label=weavevolumes)
         [ -n "$VOLUME_CONTAINERS" ] && docker rm -v $VOLUME_CONTAINERS  >/dev/null 2>&1 || true
+        rm -f $HOST_ROOT/var/lib/weave/weave-netdata.db >/dev/null 2>&1 || true
```

Comment:
What you are missing is that the directory shows up inside the container at `mountPath: /weavedb`. The directory `/var/lib/weave` is mapped to that, not concatenated.